### PR TITLE
added Home

### PR DESCRIPTION
**Pull request by Asuraking1n**

### CODE REVIEW

File: html/home.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
...xZ96n3++VSHbYwt7REMmfdPxEW5cH1bjgY1pZ+qRyt7oHhJzBHiIdHm2ltyX4w08jf3Ee+eYOoFVibbB5XgKxIN8z5fORiQNWv2x2q4Osk8owXOxT7pKAg7cp4eE9IPDziCDcAEXOoWuTykUhhNN7Iag9JVW+fowQN40xc+GXnOGw/V3FuwVcPXJIuP9J1Ft+kwtbjeaR2edT8Rhqpr1wq3RlC6QZ7sVzbRuoFgdpkDXa50YSKOJUZJpU6htqVu9TJOwA6Y5uJmQqjffln7p9AdaujjiMJWpAXZkuo3upDoP3KBMDLBcjkd22aiUkuSMlPjYTwaV9g8/lFekvqBPhGyzbrcyPheKLnq1VKYqg+kMq1O+WwuA23cTN+nzhhqjKyvl3SDlwIM+jgb5785lXlR7AnXYE232F5g1d9Ik6RDNdja1sydh43m9zOusvUB3qF8KyAPcujsy6JJv3GCnK5OR1cshrjcSxmtcHf5iR1DcPI/OdPjepfSCH/6zON6PTYeA0w3snnQ/U7GVKyo1B6SE993sFVftEb22ADbuFyGmO67LejdDDNWI71VrA/gTIf3F/ITt4xgsIlJEpoLIihVG4D4x+ZqiEIQC8iYro2jU/8AJSpv+ZFY+ZFxJcIHP1upeBbXhwPyu6+5pCr9nuDbV6VOT3/zBnWwjRnWN7MAf/FiSODoD/cpHukcdl7qAFxCEWFyabAgjcNI3zvummwl0Z/R7L6X+5iKjH8Cog/u0paYfs8wC+tTdzveo+fghUeydZCQUeG6o4FPVw1I/nX0h8C5NpaYXA0qeVOnTT8iKn+IEkQgEIQgEw7rt0X/AE2LqAiyOS6EjIhsyAeDEibjOE7Veji1GnXA/wDExVuCvogHwZQP1SwZQal9V/AGNMx3HxsI4am7bGaRao2ggZ3z7iKXc8lUEnylqFK5AOrVzvPpPDJZEBN7KovvsALzI+qHUSvUqq+JptSpIUbRcDSq+t3bA3UXAvpAGxtbaNgmVEIQgEIQgEIQgEIQgEIQgEIQgEIQgEIQgEIQgEIQgEbrUVdSrqrKwsysAVYbiDkRHIQOYpdQej1dnOHDFiTZnd0FzeyoW0QBsyyl9g8BSpDRpU6dMWtZEVBbd3QJJhAIQhAIQhAIQhAIQhAIQhAIQhAIQhAIQhAIQhAIQhAIQhAIQhAIQhAIQhA//9k=" alt="">
```
Comment (Chiragmodi01) [inline html/home.html:56]: Either use any short link or store that in separate json file.

File: html/home.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <link rel="stylesheet" href="../css/home.css">
+    <link rel="stylesheet" href="https://wish-ui.netlify.app/css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css" integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Home</title>
+</head>
+<body>
+    <div class="cart-container">
+        <div class="e-navbar">
+            <div class="e-nav-logo">E-com Web Sites</div>
+            <div class="e-nav-search">
```
Comment (Chiragmodi01) [inline html/home.html:19]: Could've used nav html tag here.

File: html/home.html (hunk 1 above)
Comment (Asuraking1n) [reply html/home.html:56]: Sure, I will keep that in mind

**Review by Chiragmodi01 — COMMENTED** [review]
You can give some horizontal padding to `filter-item` class, and then use justify space-around. it'll  take space relatively,. As here it's miss-aligned under 'sort-by',
Overall looks pretty good.

File: html/home.html (hunk 2 above)
Comment (Asuraking1n) [reply html/home.html:19]: I could have but it was taking the styling of nav of component library that's why I didn't used nav

**Comment by Asuraking1n** [comment]
> You can give some horizontal padding to `filter-item` class, and then use justify space-around. it'll take space relatively,. As here it's miss-aligned under 'sort-by', Overall looks pretty good.

Sure, I will give it a try

**Pull request merged**